packages/ra-core/src/dataProvider/dataFetchActions.ts

```diff
@@ -21,6 +21,11 @@ export const fetchActionsWithArrayOfRecordsResponse = [
 ];
 export const fetchActionsWithTotalResponse = ['getList', 'getManyReference'];
 
+export const fetchActions = [
```

Comment:
I'd name it `reactAdminFetchActions`

Reply:
Done https://github.com/marmelab/react-admin/pull/8159/commits/448a8eca6631632713b4f14513c951f5af0d4b97